test/over_react/util/react_wrappers_test.dart

```diff
@@ -226,51 +226,45 @@ main() {
         });
       });
 
-      group('updates the "key" and "ref" props properly', () {
```

Comment:
I think we still want to test all of this behavior for refs; could we keep these tests in and just change `ref` to be a callback instead of a string?